plugin/build.gradle.kts

```diff
@@ -33,29 +33,104 @@ tasks.withType<KotlinJvmCompile> {
     }
 }
 
+group = "com.github.triplet.gradle"
+version = "2.0.0-beta1"
+
 gradlePlugin {
-    plugins {
-        create("play") {
-            id = "com.github.triplet.play"
-            implementationClass = "com.github.triplet.gradle.play.PlayPublisherPlugin"
-        }
+    plugins.create("play") {
+        id = "com.github.triplet.play"
+        displayName = "Gradle Play Publisher"
+        description = "Gradle Play Publisher allows you to upload your App Bundle or APK " +
+                "and other app details to the Google Play Store."
+        implementationClass = "com.github.triplet.gradle.play.PlayPublisherPlugin"
     }
 }
 
-afterEvaluate {
-    fun PomFilterContainer.removeTestDependencies() {
-        pom.whenConfigured {
-            dependencies.removeIf {
-                // Stolen from JetBrains' own sample at
-                // https://github.com/JetBrains/kotlin/blob/v1.2.50/buildSrc/src/main/kotlin/plugins/PublishedKotlinModule.kt#L86
-                InvokerHelper.getMetaClass(it).getProperty(it, "scope") == "test"
+pluginBundle {
+    website = "https://github.com/Triple-T/gradle-play-publisher"
+    vcsUrl = "https://github.com/Triple-T/gradle-play-publisher"
+    tags = listOf("android", "google-play")
+
+    mavenCoordinates {
+        groupId = project.group as String
+        artifactId = "play-publisher"
```

Comment:
@SUPERCILEX is is actually necessary? I thought it should be set by default to these values since MavenPublication has `artifactId = "play-publisher"` and `groupId = project.group`

Reply:
Yes. `mavenCoordinates` relates to the plugin portal and says we already have an existing Jcenter artifact.